content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -184,6 +184,25 @@ DaemonSet, `/var/lib/kubelet/pod-resources` must be mounted as a
 
 Support for the "PodResources service" requires `KubeletPodResources` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) to be enabled. It is enabled by default starting with Kubernetes 1.15.
 
+## Device Plugin integration with Topology Manager
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+Topology Manager is a new Kubelet component that allows reaources to be co-ordintated in a Topology aligned manner. In order to do this, the Device Plugin API was extended to include a `TopologyInfo` struct. 
+
+```gRPC
+message TopologyInfo {
+	repeated NUMANode nodes = 1;
+}
+
+message NUMANode {
+    int64 ID = 1;
+}
+```
+Device Plugins that wish to leverage the Topology Manager can send back a populated TopologyInfo struct as part of the device registration, along with the device IDs and the health of the device. The device manager will then use this information to consult with the Topology Manager to make resource assingment decisions based on Topology alignment.
+
```

Comment:
We should explicitly mention why `nodes` is a `repeated` field instead of just a single node.  We should also explicitly mention that `nodes` should be `nil` for devices that have no NUMA topology information for them (which is the default).  There was some confusion in the past about whether it should be set to `-1` instead.